site/en/404/index.md

```diff
@@ -3,7 +3,7 @@ title: 404
 layout: 'layouts/error.njk'
 ---
 
-# Aw, Snap!
+## Aw, Snap!
```

Comment:
I think this is supposed to stay an h1 but was it making the linter angry?

Reply:
I've added a disable comment on this page.